opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -12,124 +12,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import sys
+
 from logging import getLogger
-from os import environ, path
+from os import environ
 from os.path import abspath, dirname, pathsep
 from re import sub
 
 from pkg_resources import iter_entry_points
 
-from opentelemetry.environment_variables import (
-    OTEL_PYTHON_DISABLED_INSTRUMENTATIONS,
-)
 from opentelemetry.instrumentation.dependencies import (
     get_dist_dependency_conflicts,
 )
-from opentelemetry.instrumentation.distro import BaseDistro, DefaultDistro
-
-logger = getLogger(__file__)
-
+from opentelemetry.instrumentation.environment_variables import (
+    OTEL_PYTHON_DISABLED_INSTRUMENTATIONS,
+)
 
-def _load_distros() -> BaseDistro:
-    for entry_point in iter_entry_points("opentelemetry_distro"):
-        try:
-            distro = entry_point.load()()
-            if not isinstance(distro, BaseDistro):
-                logger.debug(
-                    "%s is not an OpenTelemetry Distro. Skipping",
-                    entry_point.name,
-                )
-                continue
-            logger.debug(
-                "Distribution %s will be configured", entry_point.name
-            )
-            return distro
-        except Exception as exc:  # pylint: disable=broad-except
-            logger.exception(
-                "Distribution %s configuration failed", entry_point.name
-            )
-            raise exc
-    return DefaultDistro()
+_logger = getLogger(__file__)
 
 
-def _load_instrumentors(distro):
-    package_to_exclude = environ.get(OTEL_PYTHON_DISABLED_INSTRUMENTATIONS, [])
-    if isinstance(package_to_exclude, str):
-        package_to_exclude = package_to_exclude.split(",")
-        # to handle users entering "requests , flask" or "requests, flask" with spaces
-        package_to_exclude = [x.strip() for x in package_to_exclude]
+for entry_point in iter_entry_points("opentelemetry-pre-instrument"):
+    entry_point.load()()
 
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
-        if entry_point.name in package_to_exclude:
-            logger.debug(
-                "Instrumentation skipped for library %s", entry_point.name
-            )
-            continue
+try:
+    disabled_instrumentations = set(
+        environ.get(OTEL_PYTHON_DISABLED_INSTRUMENTATIONS, "").split()
```

Comment:
They are whitespace delimited or comma delimited? The old code does `.split(",")`

Reply:
I say we use space as the delimiter, the previous code was using `,` *and* then removing the spaces with `strip()`, which is basically using two different characters as delimiters. With this approach we can make a one-step process instead:

Before:
```python
In [7]: [element.strip() for element in "abc,    def , ghi".split(",")]
Out[7]: ['abc', 'def', 'ghi']
```
After:
```python
In [8]: "abc    def  ghi".split()
Out[8]: ['abc', 'def', 'ghi']
```

Reply:
That's a bit of churn for users though, should we do it in a separate PR?

Reply:
This is resolved